.github/workflows/lit-tests.yml

```diff
@@ -0,0 +1,21 @@
+name: Run llvm-zorg's lit tests
+
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
```

Comment:
may be it is optimal to change this line `test:` -> `Zorg-Lit-tests`. This is going to be a title of the test in the `checks` section as far as I understood.

Reply:
We could add the additional jobs later, something like `Buildbot-checkconfig`, `Buildbot-pylint' or `Zorg-pylint` & etc.

Reply:
I think the top-level name is the one that's most visible in the UI, but good point we could be more explicit. I've tweaked so it all shows like below:

![image](https://github.com/user-attachments/assets/c49ca9a4-9f44-429e-b3bd-313429729394)